openedx/core/djangoapps/api_admin/admin.py

```diff
@@ -12,6 +12,8 @@ class ApiAccessRequestAdmin(admin.ModelAdmin):
     list_filter = ('status',)
     search_fields = ('user__email',)
     raw_id_fields = ('user',)
+    readonly_fields = ('user', 'website', 'reason', 'company_name', 'company_address', 'contacted', )
```

Comment:
These should not be read-only yet. Given that the user cannot edit the submission, we need some mechanism to allow for fixing errors. For example, someone types "edxorg" instead of "edx.org" as the URL.